maintenance/monitor/monitor-component-visual.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Visualizing metrics via Grafana
+description: Use open source Grafana for visualizing Calico components.
+---
+
+### Big picture
+
+Use Grafana dashboard to generate visual aids related to the health of your {{ site.prodname }}
+
+### Value
+
+Using Grafana can be beneficial for IT personel, graphs and metrics can be crucial for you in order to identify any unsual activity.
```

Comment:
"Using Grafana can be beneficial for IT personnel by providing a means to visualize metrics through graphs that can help you quickly identify unusual activity."